pridepy/download/iprox.py

```
@@ -45,6 +51,134 @@ def matches(accession: str) -> bool:
+        ):
+            logging.info(f"Skipping download as file already exists: {dest}")
+            return None
+        argv = [
+            ascp, "-QT", "-P", cls.ASPERA_PORT, "-l", maximum_bandwidth,
+            "-k", "2", source, dest,
+        ]
+        logging.info(
+            "Aspera: %s -> %s", source.replace(password, "***"), dest
+        )
+        try:
+            subprocess.run(argv, check=True, env=env)
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)